pkg/network/protocols/postgres/protocol.go

```diff
@@ -27,7 +27,8 @@ import (
 )
 
 const (
-	inFlightMap      = "postgres_in_flight"
+	// InFlightMap is the name of the in-flight map.
+	InFlightMap      = "postgres_in_flight"
```

Comment:
The exported function will be only used in the tests, then what's the benefit here?

Reply:
I just thought that it might be a smaller change that would result in fewer code changes, as you would not need to export it. However, I have no objection to leaving it like that as well. :)